src/components/Tooltips/CanvasHoverTooltip.tsx

```diff
@@ -123,7 +123,7 @@ export default function CanvasHoverTooltip(props: PropsWithChildren<CanvasHoverT
     <p key="feature_value">
       {featureName ?? "Feature"}: <span style={{ whiteSpace: "nowrap" }}>{hoveredFeatureValue}</span>
     </p>,
-    <DebugText key="object_id">Pixel value: {lastHoveredId.segId}</DebugText>,
+    <DebugText key="object_id">Labeled image value: {lastHoveredId.segId}</DebugText>,
```

Comment:
![image](https://github.com/user-attachments/assets/578e8d53-4aae-482a-bc80-8c5c0209e3f3)

I was debating between changing this to "Label ID" or "Labeled image value" or "Label ID (pixel value)". Feedback welcome here!